src/main/java/com/wavefront/sdk/common/application/ApplicationTags.java

```diff
@@ -82,7 +85,42 @@ public Builder shard(String shard) {
      * @return {@code this}
      */
     public Builder customTags(Map<String, String> customTags) {
-      this.customTags = customTags;
+      this.customTags.putAll(customTags);
+      return this;
+    }
+
+    /**
+     * Set additional custom tags from environment variables that match the given regex.
+     *
+     * @param regex Regular expression.
+     * @return {@code this}
+     */
+    public Builder tagsFromEnv(String regex) {
```

Comment:
Just realized that Application Tags does not have a unit test
Can you add a new Java class ApplicationTagsTest
https://github.com/wavefrontHQ/wavefront-sdk-java/tree/master/src/test/java/com/wavefront/sdk/common
and test this new functionality as well as current/old functionality in ApplicationTags?
Sorry for asking you to do this, but we should add this test. Can you do that?

Reply:
sure